packages/desktop-client/src/components/table.tsx

```diff
@@ -1014,6 +1020,15 @@ export const Table = forwardRef<TableHandleRef, TableProps>(
       let editing = editingId === item.id;
       let selected = isSelected && isSelected(item.id);
 
+      scrollContainer.current &&
+        getScrollWidth &&
+        getScrollWidth(
+          scrollContainer.current.offsetParent
+            ? scrollContainer.current.offsetParent.clientWidth
+            : 0,
+          scrollContainer.current ? scrollContainer.current.clientWidth : 0,
+        );
```

Comment:
Since `getScrollWidth` doesn’t return anything, `get` doesn’t feel like the right name to me. How about something like `saveScrollWidth` or `exportScrollWidth` or `setScrollWidth` etc?

also nit: prefer using an if statement here (will try to find an ESLint rule for this!)

```suggestion
      if (scrollContainer.current && getScrollWidth) {
        getScrollWidth(
          scrollContainer.current.offsetParent
            ? scrollContainer.current.offsetParent.clientWidth
            : 0,
          scrollContainer.current ? scrollContainer.current.clientWidth : 0,
        );
      }
```

Reply:
Opened #1355 for the second part.